paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationInterceptorKtx.kt

```diff
@@ -24,11 +23,5 @@ internal suspend fun IntentConfirmationInterceptor.intercept(
                 shippingValues = confirmationOption.shippingDetails?.toConfirmPaymentIntentShipping(),
             )
         }
-        null -> {
-            IntentConfirmationInterceptor.NextStep.Fail(
-                cause = IllegalStateException("Nothing selected."),
-                message = resolvableString(R.string.stripe_something_went_wrong),
-            )
-        }
```

Comment:
This was relevant when in it was `PaymentSheetViewModel` & `DefaultFlowController` but is now handled internally by both products.